libs/parallelism/execution/include/hpx/execution/traits/is_execution_policy.hpp

```diff
@@ -127,25 +138,77 @@ namespace hpx { namespace parallel { namespace execution {
     // extension:
     template <typename T>
     struct is_async_execution_policy
-      : execution::detail::is_async_execution_policy<
+      : hpx::detail::is_async_execution_policy<
             typename hpx::util::decay<T>::type>
     {
     };
 
+    template <typename T>
+    HPX_INLINE_CONSTEXPR_VARIABLE bool is_async_execution_policy_v =
+        is_async_execution_policy<T>::value;
+
     /// \cond NOINTERNAL
     template <typename T>
     struct is_rebound_execution_policy
-      : execution::detail::is_rebound_execution_policy<
+      : hpx::detail::is_rebound_execution_policy<
             typename hpx::util::decay<T>::type>
     {
     };
 
+    template <typename T>
+    HPX_INLINE_CONSTEXPR_VARIABLE bool is_rebound_execution_policy_v =
+        is_rebound_execution_policy<T>::value;
+
     // extension:
     template <typename T>
     struct is_vectorpack_execution_policy
-      : execution::detail::is_vectorpack_execution_policy<
+      : hpx::detail::is_vectorpack_execution_policy<
             typename hpx::util::decay<T>::type>
     {
     };
+
+    template <typename T>
+    HPX_INLINE_CONSTEXPR_VARIABLE bool is_vectorpack_execution_policy_v =
+        is_vectorpack_execution_policy<T>::value;
+    /// \endcond
+}    // namespace hpx
+
+namespace hpx { namespace parallel { namespace execution {
+
+    template <typename T>
+    using is_execution_policy HPX_DEPRECATED_V(1, 6,
+        "hpx::parallel::execution::is_execution_policy is deprecated, use "
+        "hpx::is_execution_policy instead") = hpx::is_execution_policy<T>;
```

Comment:
My only question here is if these need a compatibility option. With this it's no longer possible to specialize the (alias) template in `hpx::parallel::execution`. The only case of a custom policy I know of right now is in `hpx-kokkos` but that's experimental enough that there's no point in a compatibility option for that. I also suspect that very few people will have added custom policies, so I think we're fine here. However, if we do the same to `is_executor` that would most likely need more boilerplate to keep specializations in `hpx::parallel::execution` working. Do you agree with this?

Reply:
Fair points to both. My rationale was that it doesn't hurt to have the compatibility for a while. I'm happy to remove the compatibility layer. Also, I was not planning to touch the `is_executor` and friends traits (yet). We will have to reshuffle things there with the new executor interfaces anyways.

Reply:
Yeah, sorry, I didn't mean to say that we should remove these aliases. I also think we should keep them. I was thinking in the other direction, if these need a compatibility layer for *specializing* the traits, rather than just for getting the values of the traits, and I think I already answered my own question but wanted to know what you think. I think for this PR no change is needed here.

And yes, agreed on `is_executor` etc., not relevant quite yet.